CodeHub.iOS/ViewControllers/Application/SupportViewController.cs

```diff
@@ -31,7 +31,7 @@ public override void ViewDidLoad()
             var sendEmail = new BigButtonElement("Email Support", Octicon.Mail);
             var openIssues = new BigButtonElement("Existing Issues", Octicon.IssueOpened);
 
-            HeaderView.SubText = "This app is the product of hard work and great suggestions! Thank you to all whom provide feedback!";
+            HeaderView.SubText = "This app is the product of hard work and great suggestions! Thank you to all who provide feedback!";
```

Comment:
How about:

```suggestion
            HeaderView.SubText = "This app is the product of hard work and great suggestions! Thank you to all who’ve provided feedback!";
```

Reply:
Hi, Rexogamer. Your suggestion sounds good too, but my goal with this PR is to just fix the who/whom grammar issue. Your change is slightly larger in scope, so I'd prefer to leave it to a separate change.

Reply:
I don’t think this is big enough to be its own PR.

Reply:
Hi, Rexogamer. I am new to this project, so can you please clarify for me. What is your affiliation with this project? Are you offering your opinion in an official capacity, rejecting my current PR?

Reply:
No, I mean that I don’t think my *suggestion* is big enough for its own PR.

Reply:
Oh, thanks for clarifying. To me, that's the beauty of open source. If you feel it's worthwhile to spend your time on a PR for it, then it is big enough for a PR. :)